frontend/src/components/roadmap/sections/RoadmapHeader.tsx

```diff
@@ -7,34 +7,24 @@ export default function RoadmapHeader() {
 
   return (
     <div className="text-white bg-primary-darkest padding-top-4 tablet:padding-y-6">
-      <GridContainer className="padding-0">
-        <Grid row>
-          <Grid
-            tablet={{ col: 6 }}
-            className="grid-container display-flex flex-column"
-          >
-            <h1 className="font-sans-xl margin-0 tablet:font-sans-3xl tablet-lg:font-sans-3xl tablet:maxw-2">
-              {t("pageHeaderTitle")}
-            </h1>
-            <p
-              className={
-                "font-sans-sm line-height-sans-4 margin-bottom-4 tablet:font-sans-md tablet:padding-right-1 tablet:margin-bottom-0 tablet-lg:font-sans-lg"
-              }
-            >
+      <GridContainer>
+        <Grid row gap>
+          <Grid tablet={{ col: "fill" }}>
+            <h1 className="desktop-lg:font-sans-3xl">{t("pageHeaderTitle")}</h1>
+            <p className="text-balance font-sans-md tablet:font-sans-lg margin-bottom-4 tablet:margin-bottom-0">
               {t("pageHeaderParagraph")}
             </p>
           </Grid>
-          <Grid
-            className="tablet:display-flex tablet:flex-justify-end tablet:flex-align-center tablet:grid-container"
-            tablet={{ col: 6 }}
-          >
-            <Grid className="display-flex flex-justify-center flex-align-center">
+          <Grid tablet={{ col: 6 }} tabletLg={{ col: "auto" }}>
+            <Grid className="display-flex flex-justify-center flex-align-center margin-x-neg-2 tablet:margin-x-0">
               <Image
                 src="/img/roadmap-header-image.png"
-                className="minh-full width-full desktop:padding-right-2"
                 alt="Picture of person on a journey"
+                priority={false}
                 width={360}
                 height={350}
+                style={{ objectFit: "cover" }}
+                className="minh-full width-full"
               />
```

Comment:
- Use grid gap appropriately instead of manual padding classes
- Make text column fill available space
- Make img column responsive
- Use `object-fit:cover` method (as in other page heroes)